admin-dev/themes/new-theme/scss/theme.scss

```
@@ -2,6 +2,7 @@
 @import "~prestakit/dist/css/bootstrap-prestashop-ui-kit";
 @import "~bourbon/app/assets/stylesheets/bourbon";
 @import "../node_modules/eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.min.css";
+@import "../node_modules/bootstrap-colorpicker/dist/css/bootstrap-colorpicker.min.css";
```

Not absolutely certain but I think you can do it like this:

```suggestion
@import "~bootstrap-colorpicker/dist/css/bootstrap-colorpicker.min.css";
```

---

:+1:  Done